src/duplicate-search/duplicate-search-controls.module.css

```diff
@@ -1,53 +1,191 @@
-	border-color: var( --color-primary );
+.filterControls {
+	display: flex;
+	flex-direction: row;
+	column-gap: 0.625rem;
+	justify-content: flex-start;
+	align-items: center;
 }
 
-button.statusFilterOption {
-	flex: 1;
-	padding: 0.375rem;
-	border: 1px solid transparent;
-	text-align: center;
-	color: var( --color-text-primary );
+/* TODO: Move to its own app-wide styles in CSS or through a shared component */
+button.dropdownButton {
+	padding: 0.375rem 0.5625rem 0.375rem 0.5625rem;
+	display: flex;
+	align-items: center;
 	background-color: var( --color-white );
+	border-radius: 2px;
+	border: 1px solid var( --color-border-dark );
 }
 
-button.statusFilterOption:focus-visible {
-	outline: none;
-	border: 1px solid var( --color-primary-extra-dark );
+button.dropdownButton:hover {
+	background-color: var( --color-neutral-light );
 }
 
-button.statusFilterOption:hover {
-	background-color: var( --color-neutral-extra-light );
+button.dropdownButton:active {
+	background-color: var( --color-neutral );
 }
 
-button.statusFilterOption:active {
-	background-color: var( --color-neutral-light );
+button.dropdownButton:focus-visible {
+	border-color: var( --color-primary );
+	outline: 1px solid var( --color-primary );
 }
 
-button.statusFilterOption[aria-selected='true'] {
-	color: var( --color-white );
+button.dropdownButton[data-active='true'] {
 	background-color: var( --color-primary );
+	border-color: var( --color-primary );
+	color: var( --color-white );
+}
+
+button.dropdownButton[data-active='true'] .inlineIcon path {
+	fill: var( --color-white );
 }
 
-button.statusFilterOption[aria-selected='true']:hover {
+button.dropdownButton[data-active='true']:hover {
 	background-color: var( --color-primary-dark );
+	border-color: var( --color-primary-dark );
 }
 
-button.statusFilterOption[aria-selected='true']:active {
+button.dropdownButton[data-active='true']:active {
 	background-color: var( --color-primary-extra-dark );
+	border-color: var( --color-primary-extra-dark );
+}
+
+.repoPopover {
+	z-index: 2;
+	background-color: var( --color-white );
+	border: 1px solid var( --color-border );
+	box-shadow: 0px 2px 3px -1px rgba( 0, 0, 0, 0.1 );
+	padding: 1rem;
+	overflow: auto;
+	width: 20rem;
+	height: auto;
+	min-height: 16rem;
+	min-width: 12rem;
+	resize: both;
+	display: flex;
+	flex-direction: column;
+}
+
+.repoPopoverHeader {
+	flex-shrink: 0;
+	margin-bottom: 1rem;
+}
+
+.repoPopoverBody {
+	flex-grow: 1;
+	overflow: auto;
+}
+
+.repoPopoverManualModeBody {
+	/* This height tends to cut a filter in half, which is good for showing scroll ability */
+	height: 23rem;
+}
+
+.repoPopoverFooter {
+	flex-shrink: 0;
+	padding: 1rem 0 0 0;
+	margin: 1rem 0 0 0;
+	border-top: 1px solid var( --color-border );
+}
+
+.repoFilterButtonWrapper {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-end;
+	column-gap: 0.75rem;
+}
+
+.repoFilterModeControl {
+	margin: 0 auto;
+}
+
+.repoFilterModeDescription {
+	font-size: var( --font-body-small );
```

Comment:
**apologies (sincere):**
🖐️ You can blame me for this, Dan! I increased the description font size in my latest Figma update without telling you 🙇 

For context, once we started using the "simplified" manual repo selection and added the "Select all" text under the description, it was starting to look strange having the two different sizes. In hindsight, I should've had it at `1rem` to begin with, because it is quite necessary information.